README.md

```diff
@@ -0,0 +1,109 @@
+
+
+### Workflow
+1. Input: BCDC-Metadata
+2. Data Wrangling: dashboard_data_wrangling.ipynb
+    Output: dash_count_df.csv
+3. Dashboard script: tast_dash_counts.py (takes dash_count_df.csv as input)
+    Output: Dashboard page with visualizations
+
+### Business logics:
+
+[dashboard_logics.pdf](https://github.com/AllenInstitute/BCDC-Dashboards/files/11033385/dashboard_logics.pdf)
+
+Currently, we are aiming to count the below units:
+1. Number of samples received for each sample type per group - for example, how many brain/cell related samples did we receive per quarter; or for each grant?
+2. Number of donors/subjects per group - for example, how many donors/subjects per quarter/modality/technique?
+3. Number of subspecimens (brains, cells, tissue samples, etc.) processed per quarter/grant/technique/etc.
+
+While counting, the groups and categories will be collapsed into similar terminologies - for example, 'whole cell' and 'cell body' both mean the same thing, and as a result, they will be collapsed together under 'cells' and their counts will be counted towards the same category. 
+
+1. Logics to collapse subspecimen types for counts:
+
+```
+specimen_count_map = {
+    'brains' : ['whole brain',
+                'brain', 'brains'], # fix the plural
+    'cells' : ['whole cell', 
+               'cell', 'cells', 'cell body', 'cell suspension', 'cell nucleus', 'nuclei', 'resconstruction', 'cell in slice'],
+    'nucleus' : ['cell nucleus', 
+                 'nuclei'], # collapsed nucleus and cells above in 'cells' - however, do we still want a separate 'nucleus' count?
+    'library' : ['library'],
+    'tissue samples' : ['brain hemisphere', 
+                        'brain region', 
+                        'brain section set', 
+                        'brain slice', 
+                        'library', 
+                        'spinal cord slice',
+                        'tissue sample']   
+}
+```
+Since specimen types such as 'brain' and 'whole brain' mean the same thing, they are being collapsed into the same categories and their according numbers of processed specimens will be summed and counted towards the same category.
+
+2. Logics to collapse species for counts:
+
+```
+species_map = {
+    'primates' : ['chimpanzee', 'small-eared galago', 'western gorilla', 'green monkey', 'pig-tailed macaque',
+                 "ma's night monkey", 'rhesus macaque', 'bolivian squirrel monkey',
+       'crab-eating macaque'],
+    'humans' : ['human'], ## mostly mice and humans
+    'mice' : ['mouse'],
+    'marmoset': ['marmoset'],
+    'small mammals': [
```

Comment:
Many of the primates are also small mammals, so this could be confusing. Maybe `non-primate small mammals` or something?

Reply:
changed this to 'other mammals' - opening a new PR soon